java/org/apache/tomcat/util/net/SSLHostConfig.java

```diff
@@ -796,9 +796,6 @@ public static String adjustRelativePath(String path) throws FileNotFoundExceptio
             newPath = System.getProperty(Constants.CATALINA_BASE_PROP) + File.separator + newPath;
             f = new File(newPath);
         }
-        if (!f.exists()) {
```

Comment:
Why remove this ?

Reply:
Note the first line of the PR and the prefix on the commit.

Reply:
Yes ok.

Reply:
I left the commits intentionally for those who like to test this PR in action.